engine/kafka/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -41,3 +46,21 @@ object KafkaConfig {
   def parseProcessObjectDependencies(processObjectDependencies: ProcessObjectDependencies): KafkaConfig =
     parseConfig(processObjectDependencies.config, defaultGlobalKafkaConfigPath)
 }
+
+case class TopicsExistenceValidationConfig(enabled: Boolean, validatorConfig: Option[CachedTopicsExistenceValidatorConfig] = None) {
+  def getValidatorConfig = validatorConfig.getOrElse(CachedTopicsExistenceValidatorConfig.DefaultConfig)
```

Comment:
can't we just put DefaultConfig as default argument?

Reply:
done